lib/src/test/kotlin/WeatherSensorService/WeatherSensorServiceTests.kt

```diff
@@ -0,0 +1,19 @@
+//
+// SpinDance
+// Weather Sensor Service
+//
+// Copyright © 2022 SpinDance. All rights reserved.
+//
+
+package WeatherSensorService
+import WeatherSensorService
+import WeatherSensorReaderType
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+final class WeatherSensorServiceTests {
+    /// Simply verify the singleton supports a ``WeatherSensorReaderType``
```

Comment:
I don't think the double backtick thing has the same effect here in Java land. Might as well remove those, or figure out how to add a link doc comment style (not really necessary).